src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -77,8 +77,8 @@
 
```

Comment:
I believe we want to retire this file itself. It seems like you didn't remove because `messageProvider` in MainGapicProviderFactory is still using it. Consider creating another PR for its substitute. It seems Ruby already removed the GapicContext, you can go in the same path.